Example/Tests/Classes/LeanplumTest.m

```diff
@@ -123,6 +123,48 @@ - (void) test_simple_development_start
     XCTAssertNotNil([Leanplum deviceId]);
 }
 
+/**
+ * Tests a simple development start with gzip response and validate success.
+ */
+- (void) test_simple_development_gzip_start
```

Comment:
You're verifying the request parameters here in your test. What you're missing is at least 2 use cases for the response:
1. compressed response, with the header `content-encoding: gzip`, should prompt the network module to turn the response into plain text and make it available to the start response handler
2. uncompressed response, without the `content-encoding` header should make the response available to the response handler untouched

Reply:
Ignore this please. I'm actually asking you to test NSURLSession itself!!